src/routes/api/webhooks/github.ts

```diff
@@ -0,0 +1,300 @@
+import * as crypto from 'node:crypto'
+import { MessageEmbed } from 'discord.js'
+
+function createReleaseMessage(payload) {
+  let embed = new MessageEmbed()
+  embed.setTitle(`[${payload.repository.full_name}] ${payload.release.name}`)
+  embed.setColor('#ff9900')
+  embed.setDescription(payload.release.body)
+  embed.setURL(payload.release.html_url)
+  embed.setAuthor({
+    name: payload.repository.full_name,
+    url: payload.release.author.html_url,
+    iconURL: payload.release.author.avatar_url,
+  })
+  return {
+    content: 'New Release!',
+    tts: false,
+    embeds: [embed],
+  }
+}
+
+// https://gist.github.com/stigok/57d075c1cf2a609cb758898c0b202428
+function verifyGithubWebhookEvent(payloadbody, signature256: string) {
+  const token = process.env.GITHUB_WEBHOOK_SECRET
+  const sig = Buffer.from(signature256 || '', 'utf8')
```

Comment:
If `sig256` is null/empty here we want to respond with a 400 code or something similar as well, since this is the only way we can verify the payload comes from GitHub.